pkg/controller/scyllacluster/resource_test.go

```diff
@@ -925,3 +927,227 @@ func TestStatefulSetForRack(t *testing.T) {
 		})
 	}
 }
+
+func TestMakeIngresses(t *testing.T) {
+	basicScyllaCluster := &scyllav1.ScyllaCluster{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "basic",
+			UID:  "the-uid",
+		},
+		Spec: scyllav1.ScyllaClusterSpec{
+			Datacenter: scyllav1.DatacenterSpec{
+				Name: "dc",
+				Racks: []scyllav1.RackSpec{
+					{
+						Name: "rack",
+						Storage: scyllav1.StorageSpec{
+							Capacity: "1Gi",
+						},
+					},
+				},
+			},
+		},
+	}
+
+	newIdentityService := func(name string) *corev1.Service {
+		return &corev1.Service{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: name,
+				Labels: map[string]string{
+					naming.ScyllaServiceTypeLabel: string(naming.ScyllaServiceTypeIdentity),
+				},
+			},
+		}
+	}
+
+	newMemberService := func(name, hostID string) *corev1.Service {
+		return &corev1.Service{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: name,
+				Annotations: map[string]string{
+					"internal.scylla-operator.scylladb.com/host-id": hostID,
+				},
+				Labels: map[string]string{
+					"scylla-operator.scylladb.com/scylla-service-type": "member",
+				},
+			},
+		}
+	}
+
+	newIngressRule := func(host, serviceName, servicePort, path string, pathPrefix networkingv1.PathType) networkingv1.IngressRule {
+		return networkingv1.IngressRule{
+			Host: host,
+			IngressRuleValue: networkingv1.IngressRuleValue{
+				HTTP: &networkingv1.HTTPIngressRuleValue{
+					Paths: []networkingv1.HTTPIngressPath{
+						{
+							Path:     path,
+							PathType: &pathPrefix,
+							Backend: networkingv1.IngressBackend{
+								Service: &networkingv1.IngressServiceBackend{
+									Name: serviceName,
+									Port: networkingv1.ServiceBackendPort{
+										Name: servicePort,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		}
+	}
+
+	newIngress := func(name, ingressType string, ingressClassName string, annotations map[string]string, rules ...networkingv1.IngressRule) *networkingv1.Ingress {
+		return &networkingv1.Ingress{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: name,
+				Labels: map[string]string{
+					"app":                          "scylla",
+					"app.kubernetes.io/name":       "scylla",
+					"app.kubernetes.io/managed-by": "scylla-operator",
+					"scylla/cluster":               "basic",
+					"scylla-operator.scylladb.com/scylla-ingress-type": ingressType,
+				},
+				Annotations: annotations,
+				OwnerReferences: []metav1.OwnerReference{
+					{
+						APIVersion:         "scylla.scylladb.com/v1",
+						Kind:               "ScyllaCluster",
+						Name:               "basic",
+						UID:                "the-uid",
+						Controller:         pointer.BoolPtr(true),
+						BlockOwnerDeletion: pointer.BoolPtr(true),
+					},
+				},
+			},
+			Spec: networkingv1.IngressSpec{
+				IngressClassName: pointer.String(ingressClassName),
+				Rules:            rules,
+			},
+		}
+	}
+
+	tcs := []struct {
```

Comment:
nit: `tt` (aka `test table`, to keep conventions)

Reply:
fixed